src/@carbon-info/views/Features/Features.tsx

```diff
@@ -327,12 +327,14 @@ const Features: React.FC = () => {
                       <span style={{ color: theme.palette.text.primary }}>{item.subheader}</span>
                       &nbsp;{item.description}
                     </Typography>
-                    <CTAButton
-                      text="Read Our Docs"
-                      link={item.ctaLink}
-                      textClassName={classes.ctaButtonText}
-                      iconClassName={classes.ctaButtonIcon}
-                    />
+                    <Box className={classes.ctaButtonBox}>
```

Comment:
Instead of wrapping it in a box to style, can just style it directly using the `buttonClassName` from CTAButton